anndata/tests/test_gpu.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+@pytest.mark.gpu
+def test_gpu():
```

Comment:
Do you want to keep this? It looks like a dummy

Reply:
It's testing that the gpu tests aren't being run when cupy isn't installed